backend/app/controllers/api/v1/participant/studies_controller.rb

```diff
@@ -5,7 +5,7 @@ class Api::V1::Participant::StudiesController < Api::V1::BaseController
   before_action :set_study, only: [:launch, :land, :stats]
 
   def index
-    launched_studies = current_user.launched_studies.includes(:stages, study: [:researchers])
+    launched_studies = current_user.active_launched_studies.includes(:stages, study: [:researchers])
 
```

Comment:
I don't think this is correct.

The `launched_studies` previously included all studies the user had ever launched, and this change will only include them if they're active, so should probably be renamed to `available_launched_studies`

This is then added to the `unlaunched_studies` below which is also poorly named (should probably be `available_studies`) which is a list of all the studies that are available. 

Since the 2nd variable already includes all available studies, the "available_launched_studies" will already be included in it.

I think what we intend to happen here is to include studies that are not available but have been *completed*, right?  In that case the 1st query should be rewritten to be: `completed_studies = current_user.launched_studies.includes(:stages, study: [:researchers]).complete` 

Its likely I've misread something though, and I'm not sure ☝️ is very clear 😁

Reply:
oh and I was probably the one who named all this, so that's on me 😭